http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala

```diff
@@ -769,11 +778,16 @@ class DataprocInterpreter[F[_]: Parallel](
       _ <- logger.debug(ctx.loggingCtx)(s"Resolved machine type: ${resolvedMachineType.toString}")
       total = MemorySize.fromMb(resolvedMachineType.getMemoryMb.toDouble)
     } yield {
-      // total - dataproc allocated - welder allocated
-      val dataprocAllocated = config.dataprocConfig.dataprocReservedMemory.map(_.bytes).getOrElse(0L)
-      val welderAllocated = config.welderConfig.welderReservedMemory.map(_.bytes).getOrElse(0L)
-      val result = MemorySize(total.bytes - dataprocAllocated - welderAllocated)
-      RuntimeResourceConstraints(result)
+      // For dataproc, we don't need much memory for Jupyter.
+
+      // Hail reserves about 80% of memory available to the machine to the JVM.
+      val sparkMemoryConfigRatio = config.dataprocConfig.sparkMemoryConfigRatio.getOrElse(0.8)
+      // We still want a minimum to run Jupyter and other system processes.
+      val minRuntimeMemoryGb = 4
```

Comment:
This should also be read from the configs no?

Reply:
Oh yes! I forgot to wrap that part up.